config/jobs/periodic/kubeflow/kubeflow-e2e-periodics.yaml

```diff
@@ -0,0 +1,87 @@
+periodics:
+  - name: periodic-kubeflow-pipeline-e2e-test-ppc64le
+    cluster: kubeflow-ppc64le-cluster
+    labels:
+      preset-kubeflow-powervs: "true"
+    decorate: true
+    decoration_config:
+      gcs_configuration:
+        bucket: ppc64le-kubernetes
+        path_strategy: explicit
+      gcs_credentials_secret: gcs-credentials
+    cron: "0 12 * * *"     
+    extra_refs:
+      - base_ref: master
+        org: kubeflow
+        repo: pipelines
+      - base_ref: master
+        org: ppc64le-cloud
+        repo: kubetest2-plugins
+      - base_ref: master
+        org: kubernetes-sigs
+        repo: kubetest2
+        workdir: true
+    spec:
+      containers:
+        - image: quay.io/powercloud/all-in-one:0.6
+          resources:
+            requests:
+              cpu: "4000m"
+            limits:
+              cpu: "4000m"
+          command:
+            - /bin/bash
+          args:
+            - -c
+            - |
+              set -o errexit
+              set -o nounset
+              set -o pipefail
+              set -o xtrace
+
+              export PATH=$GOPATH/bin:$PATH
+              export GO111MODULE=on
+
+              make install
+              make install-tester-ginkgo
+
+              pushd ../../ppc64le-cloud/kubetest2-plugins
+              make install-deployer-tf
+              popd
+
+              TIMESTAMP=$(date +%s)
+              K8S_BUILD_VERSION=$(curl https://storage.googleapis.com/k8s-release-dev/ci/latest.txt)
+
+              # kubectl needed for the e2e tests
```

Comment:
See if kubectl is needed for your tests.

Reply:
It's not needed for this particular set of tests